### ISSUE TITLE
Modules dependencies

### ISSUE THREAD

**Issue by fk1blow** [aspnetboilerplate/abp-ng2-module#37]
Looking at the `AbpSessionService` and found references to `abp`, which is "something" not present in the import statement. 
Is the `abp` a global loaded script alongside abp angular? Is there a reason for loading those script that way?

Thanks

**Comment by ismcagdas** [comment]
@fk1blow abp-ng2-module depends on abp-web-resources and it contains abp.js which is required for abp-ng2-module to work.

Actually we started to write abp-ng2-module to add basic angular 2 support to abp. For now it doesn't follow some practices of angular module development.

**Comment by fk1blow** [comment]
yeah, it s really hard to follow on global (window) dependencies, scattered throughout the code. For the time being, we just need the permissions/roles to work but the abp-resource loads some css too.... 
Can we just use permissions/roles withtout the abp-resource?

**Comment by ismcagdas** [comment]
> Can we just use permissions/roles withtout the abp-resource?

No, because they use functions from abp-web-resources, see https://github.com/aspnetboilerplate/abp-ng2-module/blob/master/src/auth/permission-checker.service.ts#L7.
